Changelog

```diff
@@ -1,5 +1,8 @@
 # Version 999.999.999
 
+New features
+  - Add LMove command
+
```

Comment:
```suggestion
```
you don't need to modify this file, i will do this when we release new version.

Reply:
@ShooterIT I removed the record from Changelog. Sorry, didn't see that your suggestion can be committed just by clicking one button.